src/devices/controller.c

```diff
@@ -260,9 +148,20 @@ int32_t controller_print(controller_id_e_t id, uint8_t line, uint8_t col, const
 }
 
 int32_t controller_clear_line(controller_id_e_t id, uint8_t line) {
-	static const char* clear = "                   ";
-	kassert(strlen(clear) == CONTROLLER_MAX_COLS);
-	return controller_print(id, line, 0, clear);
+	uint8_t port;
+	CONTROLLER_PORT_MUTEX_TAKE(id, port);
+	static char* clear;
```

Comment:
```suggestion
	static char* clear;
```
this is unused and can be removed